Improving Documentation (#175)
Impostor/Impostor: docs/TROUBLESHOOTING.md

# Troubleshooting
If you're reading this, something went wrong.

> Don't worry though, as this is the most thorough guide to help you~~(and to help us)~~!

Reviewer: Space before ~~ formatting

> Don't worry though, as this is the most thorough guide to help you ~~(and to help us)~~!

Reviewer 2: `~~(and to help us)~~` wtf?

Author: To help us not having to answer more than once the same questions, I can remove this too.

Reviewer 2: Remove it, looks unprofessional.

## `./Impostor.Server: line 1: ELF: not found` (plus other errors)
No idea where you got that system. But we clearly do **NOT** support it.

## `cannot execute binary file: Exec format error`

> Please check that you have downloaded the right version of Impostor, as we mantain two CPU architectures(x64 and ARM).  
> Unless you are running Impostor on some SBC (Single-Board Computer) like the Raspberry Pi, you most likely want the x64 version.

Reviewer: ```suggestion
Please check that you have downloaded the right version of Impostor, as we mantain two CPU architectures (x64 and ARM).  
Unless you are running Impostor on a SBC (Single-Board Computer), like the Raspberry Pi, you most likely want to use the x64 version.
```

## `./Impostor.Server: Permission denied`

> This is an error about Linux file permissions.  
> You can solve this by doing: `chmod +x Impostor.Server  
> (Note: only chmod when you trust the software. Linux blocks downloaded executables by default for a reason)

Reviewer: ```suggestion
This is an error related to Linux file permissions.  
You can solve this by doing: `chmod +x Impostor.Server`
_(Note: only run `chmod` when you trust the software. Linux blocks downloaded executables by default for a reason)_
```

> _(Note: only run `chmod` when you trust the software. Linux blocks downloaded executables by default for a reason)_

Reviewer 2: More because when you download it, it doesn't have the executable bit set. You can download files that have this already set.

Author: Do I really need to specify that? (are we trying to keep this a blob of text that people will most likely ignore)
Generally files dont hold their permissions

Reviewer 2: No, I'm just saying you are putting inaccurate information here.

Author: Fixed

## `You are using an older version of the game`
You are using an older version of Impostor. The game does not really check who is outdated and blames it on the user.  
Make sure you got the latest working version of Impostor (probably in AppVeyor, not Github).

## `You disconnected from the server. Reliable Packet 1 ...`
Please double-check that you have followed the [Server Configuration](Server-configuration.md) correctly.  

> **NOTE: Your public ip does not start with `127` nor `192`**  

Reviewer 2: or <https://en.wikipedia.org/wiki/Reserved_IP_addresses>.

Author: Could be, but I summed it up because those 2 cases are the most common ones

> Please check that you have followed the [Server Configuration](Server-configuration.md).  
> **REMEMBER: Your public ip does not start with `127` nor `192`**  
> Also check if the firewall has the port Impostor is listening on *unlocked* and that you port-forwarded with UDP (can be TCP/UDP, doesnt matter.)  

Reviewer: ```suggestion
Please double-check that you have followed the [Server Configuration](Server-configuration.md) correctly.  
**NOTE: Your public ip does not start with `127` nor `192`**  
Also check if the port Impostor is listening on is *unlocked* in your firewall and that you correctly port-forwarded it with UDP (can be TCP/UDP, doesn't matter.)  
```

> Also check if the port Impostor is listening on is *unlocked* in your firewall and that you correctly port-forwarded it with UDP (can be TCP/UDP, doesn't matter.)  

Reviewer 2: Unlocked?

Author: Changed to "opened"

Reviewer 2: Opened? Just change the text to 

> Also check if the port Impostor (`ListenPort`) is listening on is correctly port-forwarded for UDP (or TCP/UDP).

Author: Its important to remind people to check their firewalls, as they might be the issue

## `Could not load file or assembly...`
Please check that you only have **working** plugins in the `plugins` folder.  

> Both "not-plugins" and badly written plugins will cause this error.

Reviewer: ```suggestion
This error can be caused by non-plugin files or plugins that are not working correctly.
```
